### PR TITLE
live: mention HA limitations in administration docs

### PR DESCRIPTION
**Pull request by FZambia**
**Which issue(s) this PR fixes**:

Addresses https://github.com/grafana/grafana/issues/35448#issuecomment-872224175

### CODE REVIEW

**Review by ryantxu — APPROVED** [review]

File: docs/sources/administration/set-up-for-high-availability.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ Grafana will now persist all long term data in the database. How to configure th
 
 Currently alerting supports a limited form of high availability. [Alert notifications]({{< relref "../alerting/old-alerting/notifications.md" >}}) are deduplicated when running multiple servers. This means all alerts are executed on every server but alert notifications are only sent once per alert. Grafana does not support load distribution between servers.
 
+## Grafana Live
+
+Grafana Live works with limitations in highly available setup, see details in [Grafana Live docs]({{< relref "../live/live-ha-setup.md" >}}).
```
Comment (achatterjee-grafana) [inline docs/sources/administration/set-up-for-high-availability.md:29]:
```suggestion
Grafana Live works with limitations in highly available setup. For redetails, refer to the [Grafana Live documentation]({{< relref "../live/live-ha-setup.md" >}}).
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review.

File: docs/sources/administration/set-up-for-high-availability.md (hunk 1 above)
Comment (FZambia) [reply docs/sources/administration/set-up-for-high-availability.md]: Thanks, pushed a fix manually due to typo in suggestion.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

**Pull request merged**